spec/integration/common.js

```diff
@@ -72,8 +80,14 @@ Then('I should see {string} in {string}', assertInputValueEQ);
 Then('I should not see {string} in {string}', assertNotInputValueNotEQ);
 
 Then('{string} should {string}', (element, expectation) => {
-  const cExpectation = expectation.replace(/\s/g, '.');
-  $(element).should(cExpectation);
+	let [shouldExpr, value] = expectation.split(':');
+	shouldExpr = shouldExpr.replace(/\s/g, '.');
+
+	if (value) {
+		return $(element).should(shouldExpr, value.trim());
+	}
+
+  $(element).should(shouldExpr);
```

Comment:
to support scenarios like ` Then "Step Title Edit Box" should "have value: Install Swiftlint 0.35"`